jmx-metrics/src/main/groovy/io/opentelemetry/contrib/jmxmetrics/MBeanHelper.groovy

```diff
@@ -95,4 +95,21 @@ class MBeanHelper {
             }
         }
     }
+
+    @PackageScope List<Tuple3<GroovyMBean, String, Object>> getAttributes(Set<String> attributes) {
+        if (mbeans == null || mbeans.size() == 0) {
+            return []
+        }
+
+        def ofInterest = isSingle ? [mbeans[0]]: mbeans
+        return [ofInterest, attributes].combinations().collect { pair ->
+            def (bean, attribute) = pair
+            try {
+                new Tuple3(bean, attribute, bean.getProperty(attribute))
+            } catch (AttributeNotFoundException e) {
+                logger.warning("Expected attribute ${attribute} not found in mbean ${bean.name()}")
```

Comment:
It seems almost bound to happen by design since we are enabling a more aggregate metric type via filtering available attributes with the added interfaces. Maybe debug level? I think the above warning could be overzealous as well so even debug level could make sense there.

Reply:
updated level

Reply:
wait actually i set the wrong level. https://docs.oracle.com/javase/7/docs/api/java/util/logging/Level.html these are the levels i see for the logger we're using here. I'm not seeing a debug level.

Reply:
Sorry, context switching.  Info or fine should be ok?